validator.py

```diff
@@ -63,25 +69,63 @@ def check_coordinates_range(list_of_coordinates):
                 raise ValueError("The y-axis must be in the range of -126 to +90")
             if coordinate[2] < -72 or coordinate[2] > 108:
                 raise ValueError("The z-axis must be in the range of -72 to +108")
+        
         return True
     
-    def test_coordinates(list_of_coordinates):
+
+    def test_coordinates(self, list_of_coordinates: List[List[int]]) -> bool:
         """
         Test the coordinates
         :param list_of_coordinates: list of coordinates
         :return: True if the coordinates are valid, False otherwise
         """
-        if Validator.check_coordinates_is_list(list_of_coordinates) == False:
+        if not Validator.check_coordinates_is_list(list_of_coordinates):
             raise ValueError("Coordinates must be in a list")
-        if Validator.check_coordinates_is_not_empty(list_of_coordinates) == False:
+        if not Validator.check_coordinates_is_not_empty(list_of_coordinates):
             raise ValueError("Coordinates must not be empty")
-        if Validator.check_coordinates_length(list_of_coordinates) == False:
+        if not Validator.check_coordinates_length(list_of_coordinates):
             raise ValueError("There must be 3 coordinates for each point")
-        if Validator.check_coordinates_is_int(list_of_coordinates) == False:
+        if not Validator.check_coordinates_is_int(list_of_coordinates):
             raise ValueError("Coordinates must be integers")
-        if Validator.check_coordinates_range(list_of_coordinates) == False:
+        if not Validator.check_coordinates_range(list_of_coordinates):
             raise ValueError("Coordinates must be in the range of the board")
         print("Coordinates are valid - all tests passed")
+        
         return True
     
 
+import pytest
+
+"""
+Tests for the Validator class
+"""
+
+def test_check_coordinates_is_int(validator):
```

Comment:
maybe add a message that the tests pass or failed (the test for the validations)